plugins/scaffolder/src/components/TemplateCard/TemplateCard.tsx

```diff
@@ -83,6 +83,7 @@ const useStyles = makeStyles(theme => ({
     top: theme.spacing(0.5),
     right: theme.spacing(0.5),
     padding: '0.25rem',
+    color: '#fff',
```

Comment:
not like "inherit" or so? if the theme alters the background it renders on, so we don't get into some dark mode problem shenanigans or so

Reply:
This is meant to fix the dark mode shenanigans. It works fine in darkmode, but it flips to dark when on light mode. It should always be light text :)